src/interval.jl

```diff
@@ -228,26 +231,26 @@ end
 Base.:⊈(a::AbstractInterval, b::AbstractInterval) = !issubset(a, b)
 Base.:⊉(a::AbstractInterval, b::AbstractInterval) = !issubset(b, a)
 
-function overlaps(a::AbstractInterval, b::AbstractInterval)
+function isoverlapping(a::AbstractInterval, b::AbstractInterval)
```

Comment:
I think `overlaps` is easier to read and has precedent with `contains` as a predicate name.

Reply:
Though `contains` is deprecated in 0.7 in favor of `occursin`.

Reply:
Same deal with plural predicate though

Reply:
I think Jeff's rule of thumb for 2-arg predicates is that it reads like English if the function is put between the arguments. To that end, `a overlaps b` makes sense.

Reply:
I can change these back. Mainly I was thinking about these after changing `less_than_disjoint` to `isless_disjoint` and was thinking these should also start with `is` for consistency.

Reply:
Votes are to leave this as `overlaps`?